src/governance.rst

```diff
@@ -0,0 +1,173 @@
+Governance
+==========
+This document outlines the policies and procedures that manage the conda-forge
+community. This document recognizes that, while packaging is deeply important
```

Comment:
Are we managing a "community", or a "project" -- I would think project is a better term.

Reply:
or maybe "organization"